tests/end2end/features/conftest.py

```diff
@@ -199,6 +199,7 @@ def run_command(quteproc, httpbin, command):
 
     command = command.replace('(port)', str(httpbin.port))
     command = command.replace('(testdata)', utils.abs_datapath())
+    command = command.replace('(tmpdir)', str(tmpdir))
```

Comment:
I did the same thing in `master` - did this somehow end up in your PR or did you do the same change? I prefer your way of creating a tempdir via `Given`, didn't think of that :smile: